ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -419,8 +419,11 @@ public boolean hasFailedOnNotFound() {
     }
     if (hasFailedOnOriginatingDcNotFound()) {
       logger.info(
-          "Terminating {} on {} due to Not_Found failure. OriginatingDC Not_Found count: {}, failure threshold: {}",
-          routerOperation, partitionId, originatingDcNotFoundCount, originatingDcNotFoundFailureThreshold);
+          "Terminating {} on {} due to Not_Found failure. Originating Not_Found count: {}, failure threshold: {},"
+              + "originatingDcOfflineReplicaCount: {}, originatingDcNameTotalReplicaCount: {},"
```

Comment:
nit: Looks like we might not need string concatenation here.

Reply:
The issue is that if I remove string concatenation, then the entire log statement has to be in the same line.

There is a tradeoff here, between how the code looks, and maybe a little bit of performance. Let me know if you want me to remove the concatenation, and if you think its ok to make the statement longer.